SYCL/ESIMD/regression/half_conversion_test.cpp

```diff
@@ -0,0 +1,97 @@
+// REQUIRES: gpu
+// UNSUPPORTED: cuda || hip
+// RUN: %clangxx -fsycl %s -o %t.out
+// RUN: %GPU_RUN_PLACEHOLDER %t.out
+//==- half_conversion_test.cpp - Test for half conversion under ESIMD_EMULATOR
+// backend -==/
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//===----------------------------------------------------------------------===//
+
+#include <CL/sycl.hpp>
```

Comment:
Please don't use CL. There is an ongoing activity to replace CL with sycl. See: https://github.com/intel/llvm-test-suite/pull/1078